hivemq-edge/src/frontend/src/extensions/datahub/components/panels/FunctionPanel.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, useMemo } from 'react'
+import { Node } from 'reactflow'
+import { Card, CardBody } from '@chakra-ui/react'
+
+import { MOCK_FUNCTION_SCHEMA } from '../../api/specs/'
+import useDataHubDraftStore from '../../hooks/useDataHubDraftStore.ts'
+import { FunctionData, PanelProps } from '../../types.ts'
+import { ReactFlowSchemaForm } from '../helpers/ReactFlowSchemaForm.tsx'
+import { datahubRJSFWidgets } from '@/extensions/datahub/components/helpers'
```

Comment:
question: I noticed some inconsistencies between import paths. Some are with an alias and some are not.

Let's check in one of our sessions on that if you like.